src/ONYXKEYS.ts

```diff
@@ -321,7 +321,7 @@ type OnyxValues = {
     [ONYXKEYS.COUNTRY_CODE]: number;
     [ONYXKEYS.COUNTRY]: string;
     [ONYXKEYS.USER]: OnyxTypes.User;
-    [ONYXKEYS.LOGIN_LIST]: OnyxTypes.Login;
+    [ONYXKEYS.LOGIN_LIST]: OnyxTypes.LoginList;
```

Comment:
Maybe could we change the type back to `Login`? `LoginList` looks so misleading to me, we are not handling any lists here. I understand that the onyx key is LOGIN_LIST, but if we have opportunity to at least name the data structure correctly we should leave it.

Reply:
Sure! ➕